sandbox/public/index.html

```diff
@@ -200,6 +200,59 @@
         clickCollectionEnabled: false,
       });
     </script>
+
+    <script nonce="%REACT_APP_NONCE%">
+      alloy("configure", {
+        clickCollectionEnabled: true,
+        clickCollection: {
+          internalLinkEnabled: true,
+          externalLinkEnabled: true,
+          downloadLinkEnabled: true,
+          sessionStorageEnabled: false,
+          eventGroupingEnabled: true,
```

Comment:
Maybe we should set `eventGroupingEnabled` to false to mirror the default "out of the box" settings a user would get?